src/overlays/actors/ovl_En_Po_Fusen/z_en_po_fusen.c

```diff
@@ -266,39 +271,38 @@ void EnPoFusen_Update(Actor* thisx, PlayState* play) {
 
 s32 EnPoFusen_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* thisx) {
     EnPoFusen* this = THIS;
-    f32 zScale;
-    f32 yScale;
-    f32 xScale;
-    s16 pad;
-    s16 zRot;
-    s16 pad2;
-    s16 xRot;
-
-    if (limbIndex == 2) {
-        zScale = (Math_CosS(this->randScaleChange) * 0.08f) + 1.0f;
-        xScale = zScale;
-        if (!zScale) {}
+
+    if (limbIndex == POE_BALLOON_LIMB_BODY) {
+        f32 zScale = (Math_CosS(this->randScaleChange) * 0.08f) + 1.0f;
+        f32 yScale;
+        f32 xScale = zScale;
+        s16 pad;
+        s16 zRot;
+        s16 pad2;
```

Comment:
Both these pads needed?

Reply:
Seems like it, removing or moving shifts the stack